src/__main__.py

```diff
@@ -130,9 +142,10 @@ def test(method, dataset, metric, min_points, graph_ratio):
 @click.option('--method', type=click.Choice(METHODS.keys()))
 @click.option('--dataset', type=str, default='*')
 @click.option('--metric', type=str, default='*')
+@click.option('--starting-depth', type=int, default=0)
 @click.option('--min-points', type=str, default='*')
 @click.option('--graph-ratio', type=str, default='*')
-def plot_results(plot, method, dataset, metric, min_points, graph_ratio):
+def plot_results(plot, method, dataset, metric, starting_depth, min_points, graph_ratio):
```

Comment:
maybe starting depth should be depth - starting depth? We won't always have starting_depth layers, will be?